scripts/alluxio/alluxio-systemd.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# alluxio home
+readonly ALLUXIO_HOME=${ALLUXIO_HOME:-"/opt/alluxio-2.8.0"}
+
+# The user who will run Alluxio, default is ubuntu
+readonly RUN_USER=${RUN_USER:-"ubuntu"}
+
+# create systemd service for master and start the service
+set_systemd_and_start_master() {
+  # create service file
+  cat > "/etc/systemd/system/alluxio-master.service" <<EOF
+[Unit]
+Description=Alluxio Master
+After=default.target
+[Service]
+Type=simple
+User=${RUN_USER}
+WorkingDirectory=${ALLUXIO_HOME}
+ExecStart=${ALLUXIO_HOME}/bin/launch-process master -c
+Restart=on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  # enable service
+  systemctl enable alluxio-master
+
+  # start service
+  systemctl restart alluxio-master
```

Comment:
this seems ok for now, but for Databricks we need to split out the creation of this and the actual starting

Reply:
Will add script to docker image